Tools/parametric_model/generate_parametric_model.py

```diff
@@ -102,14 +100,61 @@ def start_model_estimation(config, log_path, data_selection="none", plot=False,
             visual_dataframe_selector_config_dict["sub_plt3_data"].append("fisher_information_rot")
 
         model.load_dataframes(select_visual_data(model.data_df,visual_dataframe_selector_config_dict))
+        print("Interactive data selection completed.")
+
+    # Setpoint based data selection
+    elif data_selection=="setpoint":
+        print("Setpoint based data selection enabled...")
+
+        selector = data_handler.config.selection_variable
+
+        if (selector not in ['aux1', 'aux2', 'aux3', 'aux4', 'aux5', 'aux6']):
+            error_str = "Variable '{0}' is not valid for data filtering".format(selector)
+            raise AttributeError(error_str)
+
+        zero_crossings = np.where(np.diff(np.sign(data_df[selector] + (data_df[selector] == 0))))[0]
```

Comment:
This is okay for this PR, but since we have the config now we can probably set thresholds :) probably for a followup PR